docs/content/contributing/community/developer-meetings.md

```diff
@@ -0,0 +1,32 @@
+---
+title: "Developer Meetings"
+date: 2018-09-28T10:40:50-07:00
+weight: 2
+
+---
+
+We hold our weekly developer meetings every Thursday at 11am US Pacific Time. They start with a quick update on the project from the maintainers, and then we discuss things on the agenda.
+
+Absolutely everyone is invited to attend, suggest topics for the agenda, and participate! You don’t have to be a contributor or maintainer to show up and share your perspective, ideas, feelings, etc.
+
+### Meeting Details
+
+They're every Thursday at 11am to 12pm US Pacific Time and you can join the meeting here by following this link: https://zoom.us/j/193797787
+
+### Agenda
+
+The [agenda google doc](https://docs.google.com/document/d/1xpvgmR1Fq4iy1j975Tb4H_XjeXUQUOAvn0FximUzvIk/edit#heading=h.f24qbjkyfv6v)
+ is where you'll can suggest topics and look at all the topics discussed in previous meetings.
+
+If you want to suggest a topic, just add a bullet point (under the appropriate meeting day) with your first name and the topic.
+
+Here's an example of how to suggest a topic:
+
+> * (Bob) Let’s talk about something cool!
+
+> * (Alice) Let’s talk about cool thing #2!
+
+### Past Meetings
+
+Don't worry if you can't make them but want to catch up! All past development meetings available on [YouTube](https://www.youtube.com/playlist?list=PLAk08AWjk5sekD-FRjU4VVe97nltUyZ4W) for you to watch back anytime.
```

Comment:
I might suggest rewording this last sentence a bit, but not huge. `..to watch back anytime.` sounds a bit strange to me.

Reply:
I took your point into account. You're right, it's weird. I'm blaming like needing sleep or something haha.